tests/zero_code_change/test_tensorflow2_integration.py

```diff
@@ -62,8 +65,15 @@ def helper_test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
         if script_mode:
             hook = smd.KerasHook(out_dir=sim.out_dir, export_tensorboard=True)
             opt = hook.wrap_optimizer(opt)
+            run_eagerly = None
+            if is_tf_2_3():
+                # Test eager and non eager mode for v2
+                run_eagerly = eager_mode
             model.compile(
-                loss="sparse_categorical_crossentropy", optimizer=opt, metrics=["accuracy"]
+                loss="sparse_categorical_crossentropy",
+                optimizer=opt,
+                metrics=["accuracy"],
+                run_eagerly=run_eagerly,
```

Comment:
isn't this setting required in the else part below on L84?

Reply:
else runs in script mode.

Reply:
Copied the changes to L84